backend/src/controllers/studentApplySteps.js

```diff
@@ -0,0 +1,63 @@
+const studentSteps = require('../models/studentApplySteps.js')
+
+
+module.exports = {
+    getAll: async (req, res) => {
+        try {
+            const steps = await studentSteps.find()
+            res.json(steps)
+        } catch (err) {
+            res.json({ message: err })
+        }
+    },
+    getStep: async (req, res) => {
+        try {
+            const step = await studentSteps.findById(req.params.projectId)
```

Comment:
I believe this should be req.params.stepsId instead of projectId

Reply:
oh sorry about that lol. Done now